recipes/tests/test_models.py

```diff
@@ -0,0 +1,30 @@
+from datetime import timedelta
+
+from django.test import TestCase
+
+from recipes.models import Recipe
+
+
+class RecipeTestCase(TestCase):
+
+    def setUp(self):
+        Recipe.objects.create(
+            title='test_title',
+            ingredients='test_ingredients',
+            preparation_process='test_prep',
+            preparation_time=timedelta(minutes=20),
+            number_of_portions=2,
+            difficulty=1)
+
+    def test_string(self):
+        expected = 'test_title'
+        actual = Recipe.objects.get(title='test_title').__str__()
+        self.assertEquals(expected, actual)
+
+    def test_title_max_length(self):
+        max_length = Recipe._meta.get_field('title').max_length
```

Comment:
these tests are testing that the ORM is doing it's job. This isn't something we should test, because we trust Django right? :) and even at that, you're testing that something you're setting gets set to the right value, not specifically that it's being enforced where it should be

A more useful/interesting (but also still not necessary!) test would be to verify that the `max_length` is being enforced. To do that, you'd want to try and stuff a value longer than your `max_length` into the field, and test that Django rejects it. But again, this is testing that Django internals are doing what it's supposed to and that's not our job